dockstore-client/src/main/java/io/dockstore/client/cli/nested/WebserviceWorkflowClient.java

```diff
@@ -0,0 +1,105 @@
+package io.dockstore.client.cli.nested;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Supplier;
+
+import io.dockstore.client.cli.Client;
+import io.dockstore.openapi.client.model.WorkflowSubClass;
+import io.swagger.client.ApiException;
+import io.swagger.client.api.WorkflowsApi;
+import io.swagger.client.model.Workflow;
+
+import static io.dockstore.client.cli.ArgumentUtility.errorMessage;
+
+public class WebserviceWorkflowClient {
```

Comment:
A JavaDoc comment would be nice -- it's not obvious from the name that this is simplifying access to Workflows and AppTools (and auth/non-auth).